### PR TITLE
Linode Detail Header feedback

### PR DESCRIPTION
**Pull request by johnwcallahan**
## Description

- Copy icon is the size of the entire cell
- Separate AccessTable into its own component
- Fix text selection on double-click

### CODE REVIEW

File: packages/manager/src/features/linodes/LinodeEntityDetail.tsx [hunk 1]
```diff
@@ -399,14 +399,113 @@ const useBodyStyles = makeStyles((theme: Theme) => ({
...
     }
-  },
-  ipContainer: {
-    flexBasis: '35%'
-  },
-  ipContent: {
-    color: theme.cmrTextColors.tableStatic,
-    fontSize: '0.875rem',
-    lineHeight: '1rem'
+  }
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodeEntityDetail.tsx:402]: Diff is not good, sorry!

File: packages/manager/src/features/linodes/LinodeEntityDetail.tsx [hunk 2]
```diff
@@ -399,14 +399,113 @@ const useBodyStyles = makeStyles((theme: Theme) => ({
...
+      </Grid>
+
+      <Grid
+        container
+        item
+        className={classes.rightColumn}
+        direction="row"
+        justify="space-between"
+      >
+        <AccessTable
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodeEntityDetail.tsx:467]: These are the main changes to this component.

File: packages/manager/src/features/linodes/LinodeEntityDetail.tsx [hunk 3]
```diff
@@ -490,162 +593,56 @@ const useBodyStyles = makeStyles((theme: Theme) => ({
...
-  // the second IPv4 address if it exists.
-  const secondAddress = ipv6 ? ipv6 : ipv4.length > 1 ? ipv4[1] : null;
+interface AccessTableProps {
+  title: string;
+  rows: AccessTableRow[];
+  gridProps?: GridProps;
+  footer?: JSX.Element;
+}
 
+export const AccessTable: React.FC<AccessTableProps> = React.memo(props => {
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodeEntityDetail.tsx:608]: This is a new component; copied over the relevant classes from `<Body />`.

File: packages/manager/src/features/linodes/LinodeEntityDetail.tsx [hunk 4]
```diff
@@ -678,15 +675,16 @@ const useFooterStyles = makeStyles((theme: Theme) => ({
     }
   },
   detailRow: {
+    display: 'flex',
```
Comment (johnwcallahan) [inline packages/manager/src/features/linodes/LinodeEntityDetail.tsx:678]: These CSS changes fixed the text selection issue.

**Review by tiffwong — APPROVED** [review]

**Review by Jskobos — APPROVED** [review]

**Pull request merged**